src/experiments.js

```diff
@@ -153,6 +127,32 @@ export function experimentToggles(win) {
   }
 
   Object.assign(toggles_, getExperimentTogglesFromCookie(win));
+
+  if (win.AMP_CONFIG
+      && Array.isArray(win.AMP_CONFIG['allow-url-opt-in'])
+      && win.AMP_CONFIG['allow-url-opt-in'].length > 0) {
+    const allowed = win.AMP_CONFIG['allow-url-opt-in'];
```

Comment:
i don't see how this is related to the PR description..
is it a separate thing?